lib/util.js

```diff
@@ -1458,7 +1504,7 @@ module.exports = {
         // Try and set fields for originalRequest (example.request)
         thisOriginalRequest.method = item.request.method;
         try {
-          thisOriginalRequest.url = item.request.url.toString();
+          thisOriginalRequest.url = displayUrl;
```

Comment:
why is this change needed?

Reply:
Because `item.request.url` is the url with the query params of the root request but we only want the URL so that we can add the query params later according to the option `exampleParamsResolution`.